packages/next-acms/README.md

```diff
@@ -0,0 +1,17 @@
+# next-acms package
+
+This package is added as a dependency in the next-acms-basic-starter kit which will allow for shipping updates
```

Comment:
This description to me feels like it's explaining how we think about this package. I think we should instead focus on providing information that's critical to the app developers. I think we should simply say here that "This package provides tooling that helps connecting Next.js applications to Acquia CMS."